kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -109,6 +109,8 @@ public void run() {
                         try (InputStream is = definitionsFileURL.openStream()) {
                             resources.addAll(resourceInstaller.install(definitionsFileURL));
                         }
+                    } else {
+                        throw new IllegalStateException("Did not find any k8s or openshift resources configuration file.");
```

Comment:
I can see cases, where the user will not use an `env.config.url` at all (e.g. run tests against a fixed namespace, or just create a namespace to run plain `in container` tests).